examples/hexagonal_lennard_jones_potential.py

```diff
@@ -34,7 +34,7 @@
         hopping_calculator=InteractingHoppingCalculator(
             baserate=HexagonalBaseRate(rate=0.01),
             temperature=200,
-            lattice_spacing=2.5,
+            lattice_properties=(2.5, np.array([1, 0]), np.array([0.5, np.sqrt(3) / 2])),
```

Comment:
if parameters [1, 0] are not noramlized, why do we also need to specify lattice spacing as a separate parameter?